src/amenities/amenities_server/controller/AmenitiesController.cs

```diff
@@ -45,7 +45,8 @@ public ActionResult<Amenity> GetAmenityByID(int id)
     [HttpDelete("{id}")]
     public IActionResult DeleteAmenity(int id)
     {
-        //validate session
+        //todo validate session call
```

Comment:
I think [what Josh meant](https://github.com/rainclouded/Concierge/pull/12#discussion_r1779814739) was to add a `//TODO:` instead of just `todo`. `//TODO:` is often get automatically highlighted so it'd be easier to spot them out.
If it doesn't get highlighted automatically for you, try [TODO Highlight](https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight) or [TODO Tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree) on vscode